tests/torch/test_sanity_third_party.py

```diff
@@ -233,7 +235,7 @@ def setup(self, temp_folder):
         self.PATH_TO_PATCH = str(os.path.join(PROJECT_ROOT, "third_party_integration", "mmdetection",
                                               "0001-Modifications-for-NNCF-usage.patch"))
         self.VENV_MMDET_PATH = str(os.path.join(temp_folder["venv"], "mmdet"))
-        self.activate_venv = str(". {}/bin/activate".format(self.VENV_MMDET_PATH))
+        self.activate_venv = str("CUDA_HOME=/usr/local/cuda-10.2 . {}/bin/activate".format(self.VENV_MMDET_PATH))
```

Comment:
Should this be done on the docker container level instead?

Reply:
@pfinashx Could you fix cuda version in the docker container for the third-party integration test?